tests/ci/integration/ibmtpm_patch/ibmtpm-mainline-awslc.patch

```diff
@@ -0,0 +1,122 @@
+diff --git a/src/BnToOsslMath.c b/src/BnToOsslMath.c
+index a185a70..4d13267 100644
+--- a/src/BnToOsslMath.c
++++ b/src/BnToOsslMath.c
+@@ -86,6 +86,12 @@
+ #ifdef MATH_LIB_OSSL
+ #  include "BnToOsslMath_fp.h"
+
++#ifdef OPENSSL_IS_AWSLC
++    #define BN_FIELD_SIZE(a) ((a)->width)
++#else
++    #define BN_FIELD_SIZE(a) ((a)->top)
++#endif
```

Comment:
We should not add a dependency to the internals of one of our structs. Please add a proper getter method for this.